library/std/src/io/util.rs

```diff
@@ -127,6 +129,25 @@ impl BufRead for Empty {
     #[inline]
     fn consume(&mut self, _n: usize) {}
 }
```

Comment:
```suggestion
}

```
It would be good to add an empty space here.

Reply:
There wasn't an empty line between `impl Read for Empty` and `impl BufRead for Empty`. As I would put `impl Seek for Empty` among the same group of impls, I didn't put one there as well.

I don't mind either way though.